.dev/assets/shared/images/social/tiktok.svg

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   role="img"
+   aria-hidden="true"
+   viewBox="0 0 24 24"
+   version="1.1"
+   id="svg4"
+   sodipodi:docname="tiktok.svg"
+   inkscape:version="1.1.1 (1:1.1+202109281949+c3084ef5ed)"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg">
+  <defs
+     id="defs8" />
+  <sodipodi:namedview
```

Comment:
@amenk thanks for your contribution. I think this svg file could use a little cleanup though. We are really just interested in the path most of the time.

Reply:
@amenk For an example, checkout one of the svgs we have included: https://github.com/godaddy-wordpress/go/blob/master/.dev/assets/shared/images/social/github.svg?short_path=058bf33